pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -202,9 +187,73 @@ func (c *Cluster) Start() error {
 	return nil
 }
 
+// reconcile the services while requiring the active mgr to be detected
```

Comment:
```suggestion
// ReconcileServices reconcile the services while requiring the active mgr to be detected
```

Reply:
this is one of the patterns that I think is much more annoying than helpful, but ok :)